### PR TITLE
2023 updates

### PR DESCRIPTION
**Pull request by loleg**
Switched in the logo, updated the content, cleaned up files a bit.

[Screenshot](https://user-images.githubusercontent.com/31819/209099785-d71b0268-5770-407d-b9d3-3c1c5345adc0.jpg)

See also https://github.com/open-network-infrastructure/hugo-hackatheme/pull/3

### CODE REVIEW

File: content/covid.md [hunk 1]
```diff
@@ -10,23 +10,14 @@ name = "COVID-19"
...
+* Hybrid Kick-off: participants will receive the link to join an online session shortly before the start.
 * Teams: We will create breakout rooms at kick-off for participants to start teaming up.
 * Hardware: right after kick-off and during the open lab week participants will be able to pickup hardware. More details at kick-off.
 * Open lab week: the lab at Bitwäscherei will be open the whole week (16:00-22:00) as planned, while keeping safe distance. If it gets too crowded, we will assign participants to other locations.
 * Workshops: some will be switched to virtual mode. More details will follow.
-* Hackdays (30-31 Oct): in-person but with remote work encouraged. Teams will work as isolated as possible. If space is not enough, we will assign participants to other locations. Final presentations only by 1 or 2 representatives of the team.
+* Hackdays: in-person but with remote work facilitated. Teams will work as isolated as possible. If space is not enough, we will assign participants to other locations. Final presentations only by 1 or 2 representatives of the team.
```
Comment (gonzalocasas) [inline content/covid.md:20]:
```suggestion
* Hackdays: in-person but with remote work facilitated. Teams will work as isolated as possible. If space is not enough, we will assign participants to other locations.
```
I don't think we will have this problem of limiting presenters, especially in June which is low season for corona viruses.

**Review by gonzalocasas — APPROVED** [review]
LGTM!

File: content/covid.md (hunk 1 above)
Comment (loleg) [reply content/covid.md:20]: :+1: Everyone, get your fannies up on that stage!

**Comment by baxtery** [comment]
good for me, i will add the profile pic

**Review by baxtery — COMMENTED** [review]
LGTM

**Review by baxtery — COMMENTED** [review]
LGTM

**Pull request merged**